gprofiler/utils/__init__.py

```diff
@@ -378,6 +407,10 @@ def grab_gprofiler_mutex() -> bool:
     on filesystem structure (as happens with file-based locks).
     In order to see who's holding the lock now, you can run "sudo netstat -xp | grep gprofiler".
     """
+    # Bypass due to lack of Windows support
```

Comment:
Instead of adding it here - I think it's nicer if we add in `main.py` before the call to it:
```
if is_linux() and not grab_gprofiler_mutex():
```

(I think it's also time to add a helper `is_linux` which `sys.platform == linux`, in some cases it's easier to read than `not is_windows()`)